src/tests/ftest/container/basic_tx_test.py

```diff
@@ -25,75 +25,20 @@
 import os
 import time
 import traceback
-import sys
-import json
-from avocado import Test
-
-sys.path.append('./util')
-sys.path.append('../util')
-sys.path.append('../../../utils/py')
-sys.path.append('./../../utils/py')
-
-import server_utils
-import write_host_file
-import AgentUtils
-import fault_config_utils
+from apricot import TestWithServers
+
+
 from conversion import c_uuid_to_str
-from daos_api import DaosContext, DaosPool, DaosContainer, DaosApiError
+from daos_api import DaosPool, DaosContainer, DaosApiError
 
-class BasicTxTest(Test):
+class BasicTxTest(TestWithServers):
     """
     A very simple test verifying the use of transactions.  This test was
     butchered in the transition from epoch to transaction.  Need to revisit
     and make more useful when transactions are fully working.
+    :avocado: recursive
```

Comment:
what is this?

Reply:
It's a requirement of [subclassing Avocado](https://avocado-framework.readthedocs.io/en/61.0/SubclassingAvocado.html) and is apparently the defaul in more recent versions, so once we get a newer LTS, this can go away.